cms/templates/cms/program_page.html

```diff
@@ -11,7 +12,8 @@
   <div class="app-media layout-boxed">
     <div class="main-content-wrapper">
       <section class="main-content">
-        <div class="banner-wrapper banner-sub banner-program-1">
+      {% image page.background_image fill-1310x613 as tmp_photo %}
```

Comment:
i see. i'm guessing we're doing that b/c a user would be uploading their own image, which we need to apply our own sizing rules to? makes sense, but i feel like it would make more sense to do this when the image is uploaded. maybe something for a future story